### PR TITLE
added text utility and new list widgets

### PR DESCRIPTION
**Pull request by Bars92**
added text utility widget.
added mentions, my messages and my tasks widgets.

### CODE REVIEW

File: Widgets/widget-LatestMessages.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "id": "latest-messages",
+  "size": 10,
+  "fromVersion": "3.6.0",
+  "dataType": "messages",
+  "query": "(type:investigation* or type:newContentAvailable or type:mention*)",
+  "name": "Latest Messages",
+  "dateRange": {
+    "period": { "byFrom": "days", "fromValue": null }
```
Comment (orius123) [inline Widgets/widget-LatestMessages.json]: shouldn't be 7?

File: Widgets/widget-LatestMessages.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  },
+  "isPredefined": true,
+  "version": -1,
+  "category": "",
```
Comment (orius123) [inline Widgets/widget-LatestMessages.json]: I think it should be under `others`

File: Widgets/widget-MyMentions.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "fromVersion": "3.6.0",
+  "dataType": "messages",
+  "query": "type:mention*",
+  "name": "Mentions",
+  "dateRange": {
+    "period": { "byFrom": "days", "fromValue": null }
+  },
+  "isPredefined": true,
+  "version": -1,
+  "category": "",
```
Comment (orius123) [inline Widgets/widget-MyMentions.json]: should be `others`

File: Widgets/widget-MyMentions.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+  "id": "my-mentions",
+  "size": 10,
+  "fromVersion": "3.6.0",
+  "dataType": "messages",
+  "query": "type:mention*",
+  "name": "Mentions",
+  "dateRange": {
+    "period": { "byFrom": "days", "fromValue": null }
```
Comment (orius123) [inline Widgets/widget-MyMentions.json]: same. our default is 7 days

File: Widgets/widget-MyTasks.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "id": "my-tasks",
+  "size": 10,
+  "dataType": "tasks",
+  "fromVersion": "3.6.0",
+  "query": "assignee:{me} and (state:Waiting or state:inprogress or state:error)",
+  "name": "My Tasks",
+  "dateRange": {
+    "period": { "byFrom": "days", "fromValue": null }
```
Comment (orius123) [inline Widgets/widget-MyTasks.json]: same

File: Widgets/widget-TextWidget [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (orius123) [inline Widgets/widget-TextWidget]: file should be `.json`

File: Widgets/widget-TextWidget [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  "id": "text-placeholder",
+  "dataType": "incidents",
+  "name": "Text Widget",
+  "dateRange": {
+    "period": { "byFrom": "days", "fromValue": null }
+  },
+  "isPredefined": true,
+  "version": -1,
+  "category": "utilities",
```
Comment (orius123) [inline Widgets/widget-TextWidget]: Do we want a new category just for this?

File: Widgets/widget-TextWidget [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+  "widgetType": "text",
+  "params": {
+    "text": "Place your text here"
```
Comment (orius123) [inline Widgets/widget-TextWidget]: Don't we want also an example?

File: Widgets/widget-MyMentions.json (hunk 4 above)
Comment (Bars92) [reply Widgets/widget-MyMentions.json]: I want all time mentions (even if user hasn't logged in more than 7 days)

File: Widgets/widget-MyTasks.json (hunk 5 above)
Comment (Bars92) [reply Widgets/widget-MyTasks.json]: same

**Pull request merged**